### ISSUE TITLE
[Bug]: Unable to launch the software from mainland China (Doing this for a friend)

### ISSUE THREAD

**Issue by V1Htor** [misyaguziya/VRCT#33]
### Debug Tips

- [X] I'm sure I've read this project's Issues of README.

### What happened?

The software loads to half point and then closes itself.

### APP Version

latest as for 3 december 2024

### Runtime Environment

Windows 10

### Country/Region

China

### Relevant log output

```sh
No log was present. The folder is empty.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by misyaguziya** [comment]
Dear @V1Htor 

Due to the special nature of the network in China, it is difficult to verify here, so we need your help.
As far as the operation is concerned, it seems that the initialization process failed to start.
If it failed, there should be an error.log output in the folder where VRCT is stored.
Could you please share that file with us?

**Comment by V1Htor** [comment]
[error.log](https://github.com/user-attachments/files/18023892/error.log)
It tried to connect to github and couldn't do it (makes sense why it is like that). It is mandatory to have connection to github? Since VRCT uses local models, why does it need that? Either way, here you are.

**Comment by misyaguziya** [comment]
Access to github is necessary to check the latest software version and to determine whether an update is required.
Therefore, it is not expected that proxies and other influences will prevent access to github.

If you want to fix this, you could do the following.

model.py
```python
@staticmethod
def checkSoftwareUpdated():
	# check update
	update_flag = False
	try:
		response = requests_get(config.GITHUB_URL)
		json_data = response.json()
		
		version = json_data.get("name", None)
		if isinstance(version, str):
		new_version = parse(version)
		current_version = parse(config.VERSION)
		if new_version > current_version:
		update_flag = True
		print("software version", "now:", config.VERSION, "new:", version)
	except:
		pass
return update_flag
```

**Comment by V1Htor** [comment]
> Access to github is necessary to check the latest software version and to determine whether an update is required. Therefore, it is not expected that proxies and other influences will prevent access to github.


Yes, but could you write a check that if connection to github fails, it launches the program without checking for updates?

**Comment by misyaguziya** [comment]
You can have the code above implemented.
Unfortunately, we do not plan to make minor updates for this special environment in version 2.x.x.
We will implement it in version 3.0.0 if possible, but for now we ask each individual to implement it.

**Comment by misyaguziya** [comment]
Daer @V1Htor 
The above features were added in version 3.0.0.
Please check it.

**Comment by V1Htor** [comment]
![image](https://github.com/user-attachments/assets/d1186d18-ecbe-4660-9895-382209b82f87)
I can't launch it... it's forever stuck in this

**Comment by ShiinaSakamoto** [comment]
I think it's fixed now. try re-download the newest setup.exe.
Since the file may be downloaded twice, please uninstall the current version first, then download and install the latest version from Booth or Github. We apologize for the inconvenience.

*Note: You can still use the current version with administrator privileges if needed.
